sdks/builds/Makefile

```diff
@@ -1,8 +1,6 @@
 TOP=$(realpath $(CURDIR)/../..)
 -include $(TOP)/sdks/Make.config
 
-all: package
```

Comment:
We should have some other phony target as the first thing in the makefile so that `make` without any arguments prints a message and exit.

Right now I think it'll run the `configure-mono` target (which just runs autogen).